src/rdkafka_offset.c

```diff
@@ -173,7 +173,7 @@ static int64_t rd_kafka_offset_file_read (rd_kafka_toppar_t *rktp) {
 	char buf[22];
 	char *end;
 	int64_t offset;
-	int r;
+	size_t r;
```

Comment:
Yeah, noticed that as well. It's unsigned anyways, so the -1 check doesn't make too much sense.